euth_wagtail/assets/scss/core/_bootstrap-variables.scss

```diff
@@ -7,6 +7,7 @@ $gray-dark:             #2b2b2b;
 $gray:                  #818181;
 $gray-light:            #d8d8d8;
 $gray-lighter:          #f4f4f4;
+$bright:                #ffffff;
```

Comment:
for white you can always use $body-bg, it comes with bootstrap, it's not there because we don't need to overwrite it so theres no way to know unless you know bootstrap :grimacing: